### PR TITLE
Feat: set hostname properly

### PR DESCRIPTION
**Pull request by mbologna**

### CODE REVIEW

File: pillar/config.sls [hunk 1]
```diff
@@ -1 +1,3 @@
 local_domain: moio
+hostname: rpi
```
Comment (moio) [inline pillar/config.sls]: Default hostname should be raspberrypi

File: pillar/config.sls [hunk 2]
```diff
@@ -1 +1,3 @@
...
+{%- set dnsdomainname = "michelebologna.net" %}
```
Comment (moio) [inline pillar/config.sls]: Why is this needed and why is it in set form?

**Review by moio — CHANGES_REQUESTED** [review]

File: pillar/config.sls (hunk 2 above)
Comment (mbologna) [reply pillar/config.sls]: This is in a `set` form to use dnsdomainname in other variables (e.g. https://github.com/moio/raspberry-openvpn-gateway/pull/17/commits/c53ad618bb4362d8286f427c6db0c9f8301a9d97)

Comment (moio) [reply pillar/config.sls]: Please try using a plain pillar variable, thanks!

Comment (mbologna) [reply pillar/config.sls]: Done!

**Review by moio — APPROVED** [review]

**Pull request merged**